src/data/showcases.js

```diff
@@ -1148,6 +1148,15 @@ export const Showcases = [
     source: null,
     tags: ["nftproject"],
   },
+  {
+    title: "selfdriven",
+    description: 
+      "Self-driven project based learning with on-chain identity & achievements tracking.",
+    preview: require("./showcase/selfdriven.png"),
+    website: "https://slfdrvn.io",
+    source: null,
+    tags: ["educational", "identity"],
```

Comment:
```suggestion
    tags: ["identity"],
```
The `educational` tag is defined (in this JS file itself) for projects which educate people _about Cardano_.  It seems like your project encompasses more than the remaining tag `identity` although that is a good place to start.

I was thinking the term `marketplace` might be also applicable (assuming that you currently do have a functional marketplace) as a brokerage environment for educational services (?)... see others at https://developers.cardano.org/showcase/?tags=marketplace